app/models/service/containers.rb

```diff
@@ -24,6 +24,10 @@ def self.start_container!(wizard_state)
       end
     end
 
+    def self.pull_image(container)
+      container.compute_resource.create_image(:fromImage => container.repository_pull_url)
```

Comment:
Again, for standalone registries, for normal operations, index provides the url. Please see the docs.